course/migrations/0020_merge_20210721_1308.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.0.14 on 2021-07-21 20:08
```

Comment:
Merged and update model migration. Didn't want to delete and attempt to recreate as it might have conflicted with all the migrations from master.